### DISCUSSION TITLE
Color Opacity Not Working

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [ben-rogerson/twin.macro#636] [Q&A]
![image](https://user-images.githubusercontent.com/44552983/157103562-e99460d9-7b52-4edb-b847-f47a1d1a3070.png)

![image](https://user-images.githubusercontent.com/44552983/157103670-a24a7888-b4a0-4594-bd98-193d327d5ff1.png)
![image](https://user-images.githubusercontent.com/44552983/157103723-621126f4-372e-43c7-96d4-c360fae30eb8.png)

**Comment by ben-rogerson** [comment]
Currently this is output (twin.macro <= v2.8.2):

```js
tw`bg-primary/10`

// ↓ ↓ ↓ ↓ ↓ ↓

({
  "backgroundColor": "rgb(var(--theme-primary))"
});
```

The correct output should be:

```js
({
  "backgroundColor": "rgb(var(--theme-primary) / 0.1)"
});
```

Next version (twin.macro@v3.0.0) will support this - follow #589 for updates.

PS: Try pasting some code next time - this was a bit of a bugger to type out for testing.
